src/libsync/CMakeLists.txt

```diff
@@ -104,13 +104,15 @@ IF (NOT APPLE)
     )
 ENDIF(NOT APPLE)
 
+find_package(Qt5 REQUIRED COMPONENTS WebSockets)
 add_library(${synclib_NAME} SHARED ${libsync_SRCS})
 target_link_libraries(${synclib_NAME}
     "${csync_NAME}"
     OpenSSL::Crypto
     OpenSSL::SSL
     ${OS_SPECIFIC_LINK_LIBRARIES}
     Qt5::Core Qt5::Network
+    Qt5::WebSockets
```

Comment:
I guess you meant to do this in gui not in libsync? (same thing for the find_package, could be grouped inside the find_package call already in gui)

Reply:
I had erros when doing it in gui. Also if we move PushNotifications in a own class inside libsync we also need to depend on Qt websockets inside libsync, because we use it there?

Reply:
Yes, if that class is in libsync that comment obviously doesn't apply anymore. Sorry about the confusion.